test/testsyncmove.cpp

```diff
@@ -306,6 +306,49 @@ private slots:
         QCOMPARE(printDbData(fakeFolder.dbState()), printDbData(remoteInfo));
     }
 
+    void testLocalExternalStorageRenameDetection()
```

Comment:
this test is not capturing anything
I ran it without having the changes in the sync engine and the test passes successfully